Author: Joost Yervante Damad <andete@debian.org>
Description: improve error message
Bug-Debian: https://bugs.debian.org/335295

--- a/timidity/timidity.c
+++ b/timidity/timidity.c
@@ -5886,11 +5899,11 @@
 				}
 			}
 			ctl->cmsg(CMSG_FATAL, VERB_NORMAL,
-					"%s: Can't read any configuration file.\n"
+					"%s: Error reading configuration file.\n"
 					"Please check %s or %s", program_name, config1, config2);
 #else
 			ctl->cmsg(CMSG_FATAL, VERB_NORMAL,
-					"%s: Can't read any configuration file.\n"
+					"%s: Error reading configuration file.\n"
 					"Please check " CONFIG_FILE, program_name);
 #endif /* __W32__ */
 		} else
