Description:  Use "find -perm /x" instead of "find -perm +x".
Author: Andreas Metzler <ametzler@debian.org>
Bug-Debian: https://bugs.debian.org/724769

--- timidity-2.13.2.orig/autogen.sh
+++ timidity-2.13.2/autogen.sh
@@ -39,7 +39,7 @@ test -d debian && {
 	[ "$1" == "updateexec" ] && {
 		echo Generating list of executable files...
 		rm -f debian/executable.files
-		find -type f -perm +111 ! -name '.*' -fprint debian/executable.files
+		find -type f -perm /111 ! -name '.*' -fprint debian/executable.files
 	}
 
 	# Remove any files in upstream tarball that we don't have in the Debian
