Author: Reiner Herrmann <reiner@reiner-h.de>
Description: Fix spelling errors found by lintian

--- a/interface/server_c.c
+++ b/interface/server_c.c
@@ -1509,7 +1509,7 @@
 	  default:
 	    ctl.cmsg(CMSG_ERROR, VERB_NORMAL,
 		     "Undefined data 0x%02x", data_buffer[offset - 1]);
-	    send_status(401, "Wrong data is recieved (seqcmd=0x%02x)",
+	    send_status(401, "Wrong data is received (seqcmd=0x%02x)",
 			cmd);
 	    stop_playing();
 	    return 1;
--- a/interface/x_wrdwindow.c
+++ b/interface/x_wrdwindow.c
@@ -388,7 +388,7 @@
     else
       shm_screen = NULL; /* No-support other format */
     if(!shm_screen)
-      ctl->cmsg(CMSG_WARNING, VERB_VERBOSE, "X SHM Extention is off");
+      ctl->cmsg(CMSG_WARNING, VERB_VERBOSE, "X SHM Extension is off");
 #else
     shm_screen = NULL;
 #endif
@@ -1996,7 +1996,7 @@
     if(InitWin(opt) == -1)
     {
 	ctl->cmsg(CMSG_ERROR, VERB_NORMAL,
-		  "WRD: Can't open WRD window becase of error");
+		  "WRD: Can't open WRD window because of error");
 	return -1;
     }
     XMapWindow(mywin.d, mywin.w);
--- a/doc/C/timidity.cfg.5
+++ b/doc/C/timidity.cfg.5
@@ -95,7 +95,7 @@
 .BI "drumset " "[MapID2] number"
 Selects the drum set to modify.  Patch mappings that follow will
 affect this drum set.  You can indicate specific map as a target, by
-specifying any ofthe following to \fIMapID2\fP: \fBgm2drum\fP,
+specifying any of the following to \fIMapID2\fP: \fBgm2drum\fP,
 \fBsc55drum\fP, \fBsc88drum\fP, \fBsc88prodrum\fP, \fBsc8850drum\fP,
 \fBxgdrum\fP and \fBxgsfx126\fP.
 .TP
